system_metrics_collector/test/system_metrics_collector/test_linux_cpu_measurement.cpp

```diff
@@ -102,16 +100,19 @@ class TestReceiveCpuMeasurementNode : public rclcpp::Node
     // round 1 150 ms: kProcSamples[2] is collected
     // round 1 160 ms: statistics derived from kProcSamples[0-1 & 1-2] is published
     stats_calc.Reset();
-    stats_calc.AddMeasurement(ComputeCpuActivePercentage(kProcSamples[0], kProcSamples[1]));
-    stats_calc.AddMeasurement(ComputeCpuActivePercentage(kProcSamples[1], kProcSamples[2]));
+    stats_calc.AddMeasurement(ComputeCpuActivePercentage(ProcessStatCpuLine(kProcSamples[0]),
+      ProcessStatCpuLine(kProcSamples[1])));
+    stats_calc.AddMeasurement(ComputeCpuActivePercentage(ProcessStatCpuLine(kProcSamples[1]),
+      ProcessStatCpuLine(kProcSamples[2])));
```

Comment:
Imo, this would be easier to read as:
```c++
stats_calc.AddMeasurement(
  ComputeCpuActivePercentage(
    ProcessStatCpuLine(kProcSamples[0]),
    ProcessStatCpuLine(kProcSamples[1])));
stats_calc.AddMeasurement(
  ComputeCpuActivePercentage(
    ProcessStatCpuLine(kProcSamples[1]),
    ProcessStatCpuLine(kProcSamples[2])));
```

Reply:
I agree, will make those changes.